src/common/models/Projection.ts

```diff
@@ -405,37 +425,56 @@ export class Projection {
     if (this.isMetricDisabled(Metric.HOSPITAL_USAGE)) {
       return null;
     }
+
+    // The ICU Capacity metric on the backend doesn't use HSA-level data for counties.
+    // We calculate it here so that the ICU Capacity metric matches the
+    // ICU actuals in this method. All counties will have HSA-level data, and all other
+    // location types will have their standard/corresponding level data.
+    const countyHsaTimeseriesIcuCapacityRatio = actualsTimeseries.map(row =>
+      this.divideICUDataWithNulls(row && row.hsaIcuBeds),
+    );
+
+    const metricSeries = this.isCounty
+      ? countyHsaTimeseriesIcuCapacityRatio
+      : metricsTimeseries.map(row => row?.icuCapacityRatio ?? null);
+
     // TODO(https://trello.com/c/bnwRazOo/): Something is broken where the API
     // top-level actuals don't match the current metric value. So we extract
     // them from the timeseries for now.
-    const icuIndex = indexOfLastValue(
-      metricsTimeseries.map(row => row?.icuCapacityRatio),
-    );
-    if (icuIndex != null && metrics.icuCapacityRatio !== null) {
-      // Make sure we don't somehow grab the wrong data, given we're pulling it from the metrics / actuals timeseries.
-      assert(
-        metrics.icuCapacityRatio === null ||
-          metrics.icuCapacityRatio ===
-            metricsTimeseries[icuIndex]?.icuCapacityRatio,
-        "Timeseries icuCapacityRatio doesn't match current metric value.",
-      );
+    const icuIndex = indexOfLastValue(metricSeries);
+
+    if (icuIndex != null) {
```

Comment:
I broke up this logic because for counties we want to check if the most recent ICU value from the HSA data timeseries we calculated above is not `null`. In order to do that, we need to access `icuIndex` (if it exists). For non-county location types, we will use `metrics.icuCapacityRatio` as we did previously.